src/file_handler/file_loader.js

```diff
@@ -249,15 +249,22 @@ cwc.fileHandler.FileLoader.prototype.loadCWCFile = async function(data,
 
   // Handle tutorial data
   let tutorialInstance = this.helper.getInstance('tutorial');
-  if (tutorialInstance) {
-    tutorialInstance.setTutorial(file.getTutorial(userLanguage));
+  let tutorialPromise = false;
+  let tutorialSpec = file.getTutorial(userLanguage);
+  if (tutorialInstance && tutorialSpec) {
+    tutorialPromise = tutorialInstance.setTutorial(tutorialSpec);
```

Comment:
Please move this checks / logic into the tutorial instance.
If you are use `await tutorialInstance.setTutorial(tutorialSpec);` the tutorial instance could just return an resolve if there are not tutorial for the specific languages.